### ISSUE TITLE
Use `ephemeral` from LXD API rather than manually stopping and deleting instances

### ISSUE THREAD

**Issue by NucciTheBoss** [NucciTheBoss/cleantest#32]
Working towards multi-distribution support in `cleantest`, I delved more into the internals of the LXD REST API (mostly because I love LXD so much). During my research, I discovered the the LXD API supports an _ephemeral_ option; when set to true, the instance is deleted after shutdown.

Using the _ephemeral_ option would be better than how I am currently tearing down the instance - by manually issuing a synchronous command to shutdown and then delete the instance: 

https://github.com/NucciTheBoss/cleantest/blob/ed400422496ca0d2db8edc3623b6c4f71adb6e50/src/cleantest/provider/_handler/lxd_handler.py#L123-L131

TL;DR figure out the feasibility of injecting _ephemeral_ into the source configuration rather than manually stopping and deleting the instance.

### Extra resources

* [Create Instance section of the LXD REST API documentation](https://linuxcontainers.org/lxd/docs/master/api/#/instances/instances_post).

**Comment by NucciTheBoss** [comment]
Due to the design of the new LXDArchon class, LXD instances can be automatically cleaned at the end of a test run without much extra effort to determine what instances need to be deleted. As such, I am closing this issue as "Won't fix" because I do not find it to be necessary after all.
